test/modules/core/controllers/test-controller.js

```diff
@@ -1,4 +1,4 @@
-import {Timeline} from '@luma.gl/addons';
+import {Timeline} from '@luma.gl/engine';
```

Comment:
Can we re-export `Timeline` from luma core? This is the only place `@luma.gl/engine` is explicitly mentioned in deck.